assembl/tests/views_tests/test_graphql.py

```diff
@@ -0,0 +1,273 @@
+    title = models.LangString.create(title, locale_code="fr")
+    video_title = models.LangString.create(
+        u"Laurent Alexandre, chirurgien et expert en intelligence artificielle nous livre ses prédictions pour le 21e siècle.",
+        locale_code="fr")
+    video_desc = models.LangString.create(
+        u"Personne ne veut d'un monde où on pourrait manipuler nos cerveaux et où les états pourraient les bidouiller",
+        locale_code="fr")
+    thematic = models.Thematic(
+        discussion_id=discussion.id,
+        title=title,
+        identifier="survey",
+        video_title=video_title,
+        video_description=video_desc,
+        video_html_code=u"<object>....</object>",
+    )
+    test_session.add(thematic)
+    test_session.commit()
+    thematic_gid = to_global_id('Thematic', thematic.id)
+
+    res = schema.execute('query { thematics(identifier:"survey") { id, title, description, numPosts, numContributors, questions { title }, video {title, description, htmlCode} } }', context_value=graphql_request)
+    assert json.loads(json.dumps(res.data)) == {
+        u'thematics': [{u'description': None,
+                        u'id': thematic_gid,
+                        u'numContributors': 0,
+                        u'numPosts': 0,
+                        u'questions': [],
+                        u'title': u'Comprendre les dynamiques et les enjeux',
+                        u'video': {u'title': u"Laurent Alexandre, chirurgien et expert en intelligence artificielle nous livre ses prédictions pour le 21e siècle.",
+                                   u'description': u"Personne ne veut d'un monde où on pourrait manipuler nos cerveaux et où les états pourraient les bidouiller",
+                                   u'htmlCode': u"<object>....</object>",
+                                   }}]}
+
+
+def test_mutation_create_thematic_with_video(graphql_request):
+    res = schema.execute("""
+mutation myFirstMutation {
+    createThematic(titleEntries:[
+        {value:"Comprendre les dynamiques et les enjeux", localeCode:"fr"},
+        {value:"Understanding the dynamics and issues", localeCode:"en"}
+        ],
+        video: {
+            titleEntries:[
+                {value:"Laurent Alexandre, chirurgien et expert en intelligence artificielle nous livre ses prédictions pour le 21e siècle.",
+                 localeCode:"fr"},
+            ]
+            descriptionEntries:[
+                {value:"Personne ne veut d'un monde où on pourrait manipuler nos cerveaux et où les états pourraient les bidouiller",
+                 localeCode:"fr"},
+            ],
+            htmlCode:"<object>....</object>"
+        },
+        identifier:"survey") {
+        thematic {
+            title,
+            identifier
+            video {title, description, htmlCode}
+        }
+    }
+}
+""", context_value=graphql_request)
+    assert json.loads(json.dumps(res.data)) == {
+        u'createThematic': {
```

Comment:
No, it doesn't matter if it's ascii, if there is accent, I don't know. The keys and values in res.data are unicode, so I use unicode in the expected result too.